tests/MyGame/Example/Monster.php

```diff
@@ -171,8 +171,8 @@ public function getTestarrayofstringLength()
         return $o != 0 ? $this->__vector_len($o) : 0;
     }
 
-/// an example documentation comment: this will end up in the generated code
-/// multiline too
+    /// an example documentation comment: this will end up in the generated code
```

Comment:
Move this comment below into generated doc section, as Java generator does.

Reply:
It is hard to move user comments from FBS scheme to auto-generated doc-section.
Accessor generators should use `GenComment` instead of manual generation of doc-sections.
```
auto cpy_comments = filed.comments;
cpy_comments += "@Accessor annotation";
GenComment(cpy_comments, php_multiline_rules);
Generate accessor code...
```